packages/unchained-client/src/evm/parser/arbitrumBridge.ts

```diff
@@ -103,6 +107,20 @@ export class Parser implements SubParser<Tx> {
     }
 
     switch (selectedAbi) {
+      case this.arbOutboxAbi:
+        switch (txSigHash) {
+          case this.arbOutboxAbi.getFunction('executeTransaction')!.selector:
+            const amount = decoded.args.value as BigInt
+
+            return await Promise.resolve({
+              data: {
+                ...data,
+                value: amount.toString(),
```

Comment:
@kaladinlight not sure we want this one, added in case we want to use this guy at display time, but reading this again using `value` in case of ERC20s may be conceptually super wrong, happy to remove

Reply:
yeah we can remove, this data will be picked up by the parser as normal transfers or internal transfers which is sufficient for display and currently innacurate for tokens as you stated

Reply:
55834e82e9f97ea53a01fbad05728d82b61639f3